resources/Dockerfile.service

```diff
@@ -10,33 +10,24 @@ FROM ros:jazzy AS base
 WORKDIR /opt/ros/underlay
 
 ENV ROS_HOME=/tmp
-ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+ENV RMW_IMPLEMENTATION=rmw_zenoh_cpp
 
 # underlay stage: base + dependencies built
 FROM base AS underlay
 
 ADD src/intrinsic_sdk_ros /opt/ros/underlay/src/intrinsic_sdk_ros
-
-RUN git clone https://github.com/eclipse-zenoh/zenoh-plugin-dds \
-    -b release-1.0.2 \
-    --recurse-submodules \
-    /opt/ros/underlay/src/zenoh-plugin-dds
+# TODO(Yadunund): Switch to ZENOH_CONFIG_OVERRIDE after the next jazzy sync.
+COPY zenoh_config.json5 /opt/zenoh_config.json5
 
 RUN . /opt/ros/jazzy/setup.sh \
     && apt-get update \
-    && apt install -y clang-14 ros-jazzy-ament-cmake-vendor-package cargo \
-    && colcon build \
-        --continue-on-error \
-        --cmake-args -DCMAKE_C_COMPILER=clang-14 -DCMAKE_CXX_COMPILER=clang++-14 \
-        --event-handlers=console_direct+ \
-        --merge-install \
-        --packages-up-to zenoh_bridge_dds \
+    && apt install -y clang-16 ros-jazzy-ament-cmake-vendor-package cargo \
     && colcon build \
+        --cmake-args -DCMAKE_C_COMPILER=clang-16 -DCMAKE_CXX_COMPILER=clang++-16 \
         --continue-on-error \
         --event-handlers=console_direct+ \
         --merge-install \
-        --packages-up-to intrinsic_sdk \
-        --packages-skip zenoh_bridge_dds
+        --packages-up-to intrinsic_sdk
```

Comment:
I don't think we should need `cargo` or `clang-16` anymore, that should all be handled in `ros-jazzy-rmw-zenoh-cpp`.